spec/reducers/config/__snapshots__/config.spec.ts.snap

```diff
@@ -5,7 +5,7 @@ Object {
   "@@redux-saga/IO": true,
   "SELECT": Object {
     "args": Array [
-      "ELLA",
+      "CLO",
```

Comment:
Not sure why this snapshot had to be updated. I didn't touch anything related to network config. . .